qiskit_nature/properties/second_quantization/electronic/integrals/integral_property.py

```diff
@@ -173,3 +189,45 @@ def interpret(self, result: EigenstateResult) -> None:
             NotImplementedError
         """
         raise NotImplementedError()
+
+    def to_hdf5(self, parent: h5py.Group) -> None:
+        """Stores this instance in a HDF5 group inside of the provided parent group.
```

Comment:
I guess with Protocol this docstring has to be replicated in every class that implements these methods that conform to the Protocol. With ABCs they are pulled in from the parent which avoids this.

Reply:
I actually did not try that. I will do that and remove the duplicate docstrings if possible.

Reply:
Indeed, the documentation does **not** carry over.